canonical-websites/discourse-docs.snapcraft.io
# Restructure CSS to make it awesome

I've gone through the custom CSS and removed some of it, documented Vanilla fixes better, and restructured and renamed the custom local CSS to make it more semantic.

- Added comments explaining local Vanilla fixes and created and linked to bugs: https://github.com/vanilla-framework/vanilla-framework/issues/1988, https://github.com/vanilla-framework/vanilla-framework/issues/1987, https://github.com/vanilla-framework/vanilla-framework/issues/1985
- Added comment for one Discourse fix that isn't generic (`.emoji`)
- Moved locally custom CSS into `l-docs` and `p-sidenav` components

QA
--

`./run`

Go to http://0.0.0.0:8029 and check it looks and works basically exactly how it did before.
